packages/myst-config/src/site/types.ts

```diff
@@ -61,7 +61,7 @@ type ManifestProject = {
 } & ProjectFrontmatter;
 
 export type SiteManifest = SiteFrontmatter & {
-  myst: 'v1';
+  myst: string;
   id?: string;
   projects?: ManifestProject[];
   nav?: SiteNavItem[];
```

Comment:
Should this type now have `options: SiteTemplateOptions`?

Reply:
Hmm, we don't need to add it here as it comes in from `SiteFrontmatter` which now has `options?: Record<string, any>;` (and if you dig into `SiteTemplateOptions`, that type is actually just `Record<string, any>`).

I think maybe we should just delete `SiteTemplateOptions` as a type, since it only confuses the fact that it's a simple dict.

Reply:
Oh, and we should remove the `& SiteTemplateOptions` here I guess!